src/include/distributed/master_protocol.h

```diff
@@ -88,8 +88,8 @@ extern bool CStoreTable(Oid relationId);
 extern Oid ResolveRelationId(text *relationName);
 extern List * GetTableDDLEvents(Oid relationId);
 extern void CheckDistributedTable(Oid relationId);
-extern void CreateShardPlacements(int64 shardId, List *ddlEventList,
-								  char *newPlacementOwner,
+extern void CreateShardPlacements(Oid distributedTableId, int64 shardId,
```

Comment:
Can we use relationId instead of distributedTableId? Both are used in the code base, but relationId seems better in this context. 

What do you think?